web/client-api/src/main/java/io/deephaven/web/client/api/widget/plot/JsFigure.java

```diff
@@ -657,7 +656,7 @@ public Promise fetch(JsFigure figure, FetchObjectResponse response) {
                     JsTable table = new JsTable(connection, cts);
                     // never attempt a reconnect, since we might have a different figure schema entirely
                     table.addEventListener(JsTable.EVENT_DISCONNECT, ignore -> table.close());
-                    tables[p1] = table;
+                    tables[tables.length] = table;
```

Comment:
Ah the beauty of JS

Couldn't we just do the same with the promises as well?

Reply:
Could, but no need, since we know it will be sized to the full width.

I could make this more "correct in java" and count how many are tables... but it is js, so I didn't (but for a reviewer who asks for it, i will).